azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -1548,35 +1584,42 @@ public FlowRunnerEventListener() {
       final ExecutableFlow flow = flowRunner.getExecutableFlow();
       final Props props = ServiceProvider.SERVICE_PROVIDER.getInstance(Props.class);
       final Map<String, String> metaData = new HashMap<>();
-      metaData.put("flowName", flow.getId());
+      metaData.put(FLOW_NAME, flow.getId());
       // Azkaban executor hostname
-      metaData.put("azkabanHost", props.getString(AZKABAN_SERVER_HOST_NAME, "unknown"));
+      metaData.put(AZ_HOST, props.getString(AZKABAN_SERVER_HOST_NAME, "unknown"));
       // As per web server construct, When AZKABAN_WEBSERVER_EXTERNAL_HOSTNAME is set use that,
       // or else use jetty.hostname
-      metaData.put("azkabanWebserver", props.getString(AZKABAN_WEBSERVER_EXTERNAL_HOSTNAME,
+      metaData.put(AZ_WEBSERVER, props.getString(AZKABAN_WEBSERVER_EXTERNAL_HOSTNAME,
           props.getString("jetty.hostname", "localhost")));
-      metaData.put("projectName", flow.getProjectName());
-      metaData.put("submitUser", flow.getSubmitUser());
-      metaData.put("executionId", String.valueOf(flow.getExecutionId()));
-      metaData.put("startTime", String.valueOf(flow.getStartTime()));
-      metaData.put("submitTime", String.valueOf(flow.getSubmitTime()));
+      metaData.put(PROJECT_NAME, flow.getProjectName());
+      metaData.put(SUBMIT_USER, flow.getSubmitUser());
+      metaData.put(EXECUTION_ID, String.valueOf(flow.getExecutionId()));
+      metaData.put(START_TIME, String.valueOf(flow.getStartTime()));
+      metaData.put(SUBMIT_TIME, String.valueOf(flow.getSubmitTime()));
       // Flow_Status_Changed event attributes: flowVersion, failedJobId, modifiedBy
-      metaData.put("flowVersion", String.valueOf(flow.getAzkabanFlowVersion()));
-      metaData.put("failedJobId", flow.getFailedJobId());
-      metaData.put("modifiedBy", flow.getModifiedBy());
+      metaData.put(FLOW_VERSION, String.valueOf(flow.getAzkabanFlowVersion()));
+      metaData.put(FAILED_JOB_ID, flow.getFailedJobId());
+      metaData.put(MODIFIED_BY, flow.getModifiedBy());
       // Flow_Status_Changed event elapsed time
-      metaData.put("flowKillDuration", String.valueOf(flowRunner.getFlowKillDuration()));
-      metaData.put("flowPauseDuration", String.valueOf(flowRunner.getFlowPauseDuration()));
-      metaData.put("flowPreparationDuration", String.valueOf(flowRunner.flowCreateTime));
+      metaData.put(FLOW_KILL_DURATION, String.valueOf(flowRunner.getFlowKillDuration()));
+      metaData.put(FLOW_PAUSE_DURATION, String.valueOf(flowRunner.getFlowPauseDuration()));
+      metaData.put(FLOW_PREPARATION_DURATION, String.valueOf(flowRunner.flowCreateTime));
       // FLow SLA option string
-      metaData.put("slaOptions", flow.getSlaOptionStr());
+      metaData.put(SLA_OPTIONS, flow.getSlaOptionStr());
+      // Flow executor type by versionSet
+      if (flow.getVersionSet() != null) {
```

Comment:
add a comment explaining when the versionSet is null.

Reply:
Added the comment.